queries.sql

```diff
@@ -23,3 +23,31 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+--  FOR PART 2 
```

Comment:
- [ ] Great job so far 👏🏻 However, you are missing some important queries you should start with👇🏻 Please add them and be sure to take a screenshot of the results of your actions. 📸


![image](https://user-images.githubusercontent.com/79058364/222088588-f6bf44dd-989d-49a1-9a0b-6f92f122c65d.png)